Unity Example/Assets/Discord RPC/Scripts/DiscordManager.cs

```diff
@@ -152,18 +153,17 @@ private static void CreateNewManager()
     }
 #endif
 
-    #endregion
+	#endregion
 
-    /// <summary>
-    /// Initializes the discord client if able to. Wont initialize if <see cref="active"/> is false, we are not in playmode, we already have a instance or we already have a client.
-    /// <para>This function is empty unless UNITY_WSA || UNITY_WSA_10_0 || UNITY_STANDALONE) && !DISABLE_DISCORD is meet.</para>
-    /// </summary>
-    public void Initialize()
-    {
+	/// <summary>
+	/// Initializes the discord client if able to. Wont initialize if <see cref="active"/> is false, we are not in playmode, we already have a instance or we already have a client.
+	/// <para>This function is empty unless UNITY_WSA || UNITY_WSA_10_0 || UNITY_STANDALONE) && !DISABLE_DISCORD is meet.</para>
+	/// </summary>
+	public void Initialize()
+	{
 #if (UNITY_WSA || UNITY_WSA_10_0 || UNITY_STANDALONE) && !DISABLE_DISCORD
 
         if (!active) return;                //Are we allowed to be active?
-        if (!Application.isPlaying) return; //We are not allowed to initialize while in the editor.
```

Comment:
We don't want this to be running in the editor generally speaking. Unity does not behave well with native pipes.
Can you confirm that it doesn't keep the Presence once its out of play mode?

Reply:
yeah fixed that